src/react/components/pages/editorPage/assetPreview.tsx

```diff
@@ -4,27 +4,15 @@ import { strings } from "../../../../common/strings";
 import { Player, ControlBar, CurrentTimeDisplay, TimeDivider,
     BigPlayButton, PlaybackRateMenuButton, VolumeMenuButton } from "video-react";
 
-/**
```

Comment:
Any particular reason we're removing the docs here?

Reply:
Absolutely not and indeed I've no idea why that happened.  Re-inserted.